src/test/java/com/iexec/sms/chain/ChainConfigTests.java

```diff
@@ -0,0 +1,263 @@
+
+    private static final int CHAIN_ID = 1;
+    private static final String HUB_ADDRESS = "0x3eca1B216A7DF1C7689aEb259fFB83ADFB894E7f";
+    private static final String NODE_ADDRESS = "https://bellecour.iex.ec";
+    private static final Duration BLOCK_TIME = Duration.ofSeconds(1);
+    private static final float GAS_PRICE_MULTIPLIER = 1.0f;
+    private static final long GAS_PRICE_CAP = 22_000_000_000L;
+
+    private Set<ConstraintViolation<ChainConfig>> validate(ChainConfig chainConfig) {
+        try (final ValidatorFactory factory = Validation.buildDefaultValidatorFactory()) {
+            return factory.getValidator().validate(chainConfig);
+        }
+    }
+
+    // region Valid data
+    static Stream<Arguments> validData() {
+        return Stream.of(
+                Arguments.of(100, true, "http://localhost:8545", "0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002", "PT0.1S", 1.0f, 11_000_000_000L),
+                Arguments.of(42, true, "https://localhost:8545", "0x0000000000000000000000000000000000000001", "PT10S", 1.0f, 22_000_000_000L),
+                Arguments.of(10, true, "https://www.classic-url.com", "0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002", "PT20S", 1.0f, 22_000_000_000L),
+                Arguments.of(1, true, "http://ibaa.iex.ec:443/test?validation=should:be@OK", "0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002", "PT5S", 1.0f, 0L),
+                Arguments.of(CHAIN_ID, true, NODE_ADDRESS, HUB_ADDRESS, BLOCK_TIME, GAS_PRICE_MULTIPLIER, GAS_PRICE_CAP)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("validData")
+    void shouldValidate(Integer chainId,
+                        boolean sidechain,
+                        String nodeAddress,
+                        String hubAddress,
+                        Duration blockTime,
+                        float gasPriceMultiplier,
+                        long gasPriceCap) {
+        final ChainConfig chainConfig = new ChainConfig(
+                chainId,
+                sidechain,
+                nodeAddress,
+                hubAddress,
+                blockTime,
+                gasPriceMultiplier,
+                gasPriceCap);
+
+        log.info("{}", chainConfig);
+        assertThatCode(() -> validate(chainConfig)).doesNotThrowAnyException();
+    }
+    // endregion
+
+    // region Invalid chain ids
+    static Stream<Integer> invalidChainIds() {
+        return Stream.of(
+                0,      // Chain id should be strictly positive
+                -1      // Chain id should be strictly positive
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("invalidChainIds")
+    void shouldNotValidateChainId(int chainId) {
+
+        final ChainConfig chainConfig = new ChainConfig(
+                chainId,
+                true,
+                NODE_ADDRESS,
+                HUB_ADDRESS,
+                BLOCK_TIME,
+                GAS_PRICE_MULTIPLIER,
+                GAS_PRICE_CAP);
+
+        log.info("{}", chainConfig);
```

Comment:
I think all those log statements can be removed. They should probably not have been committed in the past.